wrapanapi/base.py

```diff
@@ -152,7 +152,7 @@ def list_flavor(self):
         """
         raise NotImplementedError('list_flavor not implemented.')
 
-    def list_network(self):
```

Comment:
**Required**

I disagree with this change, as its a needless API change that breaks conformity of our method names.

See all other abstract methods defined on this class, they're all singular.

Reply:
Ok I did not know, I 'll change it. it just sounded wierd to me as some other stuffs were in pural too ( like _get_all_private_images, ). I ll change it. 
btw should I add also in this base.py all my other methods i implemented on providers?
 like:
list_subnet(self):
raise NotImplemented

Reply:
Oh it's definitely a bit strange, but at this point if we want to change that it's going to take a whole lot of coordination and a PR that makes the change _everywhere_ at once, and would potentially force a 3.0 release since it's such a major API change.

Reply:
@t-fischer I don't think you need to add this abstract method to the base module, since that generally means its something that the inheriting classes _should_ implement. In this case its valid for two provider types that you've added here, but not needed/relevant in other cases.

Reply:
I will also change to other methods to singular